clockify/settings.py

```diff
@@ -150,14 +160,59 @@
     "ACCESS_TOKEN_LIFETIME": timedelta(days=100),
 }
 
-SOCIAL_AUTH_FACEBOOK_KEY = config('SOCIAL_AUTH_FACEBOOK_KEY')
-SOCIAL_AUTH_FACEBOOK_SECRET = config('SOCIAL_AUTH_FACEBOOK_SECRET')
+# SOCIAL_AUTH_FACEBOOK_KEY = config('SOCIAL_AUTH_FACEBOOK_KEY')
+# SOCIAL_AUTH_FACEBOOK_SECRET = config('SOCIAL_AUTH_FACEBOOK_SECRET')
```

Comment:
commented out code shouldn't be inside main branch unless there is reason/ comment for why it should stay

Reply:
I tend to keep commented out code as reference, but it's not a good practice, I removed it, thank you.